src/main/java/com/untamedears/humbug/Versioned.java

```diff
@@ -1,8 +1,9 @@
 package com.untamedears.humbug;
 
-import org.bukkit.craftbukkit.v1_10_R1.entity.CraftHorse;
-import net.minecraft.server.v1_10_R1.EntityHorse;
-import net.minecraft.server.v1_10_R1.GenericAttributes;
+import net.minecraft.server.v1_11_R1.EntityHorse;
+import net.minecraft.server.v1_11_R1.GenericAttributes;
+
+import org.bukkit.craftbukkit.v1_11_R1.entity.CraftHorse;
 import org.bukkit.entity.Entity;
 
 public class Versioned {
```

Comment:
This is the same method.
Actually I've compared sources of 1.10.2-spigot and 1.11.2-spigot and made sure that all "magic" methods are similar.
But I could make mistake as well and probably missed something during testing, so would be good to do some testing by someone (at least fast testing)